packages/react/src/components/Popup/Popup.tsx

```diff
@@ -386,7 +386,7 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
     rtl: boolean,
     accessibility: ReactAccessibilityBehavior,
   ): JSX.Element {
-    const { align, position, offset, target } = this.props
+    const { align, position, offset, target, unstable_pinned } = this.props
```

Comment:
Why is this not added in the propTypes and props' interface? Isn't thing going to end up as unhandled prop?

Reply:
We are not using the unhandledProps in the Popup component so it won't be a problem, but still, let's add this prop in the propTypes as well

Reply:
closing for now as @layershifter will introduce a better solution, if we reopen I will add the Proptype, I missed it for sure

thanks for reviewing @mnajdova

Reply:
and we reopened it 😆 added the proptype